src/main/native/build_windows_jni.sh

```diff
@@ -51,10 +51,19 @@ if [ -z "${BAZEL_VS+set}" ]; then
 fi
 VSVARS="${BAZEL_VS}/VC/VCVARSALL.BAT"
 
-# Check if Visual Studio 2017 is installed. Look for it at the default location.
+# Check if Visual Studio 2017 is installed. Look for it at the default
+# locations.
 if [ ! -f "${VSVARS}" ]; then
-  VSVARS="C:/Program Files (x86)/Microsoft Visual Studio/2017/BuildTools/VC/"
-  VSVARS+="Auxiliary/Build/VCVARSALL.BAT"
+  VSVARS="C:/Program Files (x86)/Microsoft Visual Studio/2017/"
+  VSEDITION="BuildTools"
+  if [ -d "${VSVARS}Enterprise" ]; then
```

Comment:
Eventually (soon!) we would just kill this script in favor of building this as a normal cc_library with msvc toolchain. This is fine until then. 
Arguably, this code should look into registry (and generally duplicate the logic in cc_configure.bzl), but this is moot since this script goes away soon anyway.